client/src/app/routes/chrysalis/Addr.tsx

```diff
@@ -165,267 +114,340 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
-                                    this.state.historicOutputs.length > 0 && (
-                                        <div className="card card__secondary">
-                                            <div className="card--header card--header__space-between">
-                                                <h2>
-                                                    Historic Balances
-                                                </h2>
                                             </div>
-                                            <div className="card--content">
-                                                {this.state.historicOutputs.map((output, idx) => (
-                                                    <div key={idx} className="margin-b-m">
-                                                        <div className="card--value row middle">
-                                                            <div
-                                                                className="
-                                                                card--label card--label__no-height margin-r-s"
-                                                            >
-                                                                Message
-                                                            </div>
-                                                            <span className="card--value card--value__no-margin">
-                                                                <Link
-                                                                    to={
-                                                                        `/${this.props.match.params.network
-                                                                        }/message/${output.messageId}`
-                                                                    }
-                                                                    className="margin-r-t"
-                                                                >
-                                                                    {output.messageId}
-                                                                </Link>
-                                                            </span>
-                                                            <MessageButton
-                                                                onClick={() => ClipboardHelper.copy(
-                                                                    output.messageId
-                                                                )}
-                                                                buttonType="copy"
-                                                                labelPosition="top"
+
+                                        </div>
+                                        <table className="transaction--table">
+                                            <tr>
```

Comment:
`<tr>` cannot appear as a child of `<table>`. Add a `<tbody>`, `<thead>` or `<tfoot>` to your code to match the DOM tree generated by the browser.

Reply:
Resolved